Gruntfile.js

```diff
@@ -230,10 +230,10 @@ var _              = require('lodash'),
                 },
                 coverage_all: {
                     src: [
-                        'core/test/unit',
-                        'core/server/apps',
                         'core/test/integration',
-                        'core/test/functional'
+                        'core/server/apps',
+                        'core/test/functional',
+                        'core/test/unit'
```

Comment:
This is just reordered to show that it works. We can leave it like this or I'll change it back--doesn't matter to me.